### ISSUE TITLE
UX - Request - Mechanism for restoring an account from a key instead of a mnemonic phrase

### ISSUE THREAD

**Issue by storyaddict** [staltz/manyverse#80]
Hello :) 

TL;DR - Please allow account recovery via SSB key as well as mnemonic group of words.

I am a fan of the Manyverse and SSB in general, but one of the things keeping me from using SSB more is the inability to sync accounts across devices or app clients. Android is my daily driver, so I would like to use Manyverse most of the time, but I'm also following the Planetary project since I like what they're doing with pub servers. 

I created an account in the iOS Planetary app, and would like to add that same account to Manyverse, but I have to figure out a way to convert my key to a mnemonic before I will be able to add my Planetary account to Manyverse or vice-versa. 

Current Behaviour: New Manyverse ID creation provides only a mnemonic phrase to backup account.

Desired Behaviour: Provide the option for accessing both the SSB key and mnemonic phrase. Allow restoring account from either.

Thank you for your attention, I appreciate it! :)

**Comment by staltz** [comment]
Please do not attempt to use the same ID on multiple apps, if you do this, your account is in risk of getting "forked" in case you publish on one app, out of sync with the other, and this is an irrecoverable state, your account will be unusable forever. Thus I don't want to make it easier to use the same ID in many apps.

That said, you can convert from/to mnemonic words to SSB ID using the module https://github.com/staltz/ssb-keys-mnemonic, but please don't do this as a mechanism to have the same account in many apps.
